modules/nw-ptp-e810-hardware-configuration-reference.adoc

```diff
@@ -4,7 +4,7 @@
 
 :_mod-docs-content-type: REFERENCE
 [id="nw-ptp-wpc-hardware-pins-reference_{context}"]
```

Comment:
same as above, document id has `wpc` reference

Reply:
Updated as per the suggestion.